packages.rb

```diff
@@ -25,19 +25,21 @@
 end
 
 dep 'nodejs.bin', :version do
-  version.default!('0.10.25')
-  installs {
-    via :apt, [
-      "nodejs",
-      "nodejs-dev",
-      "nodejs-legacy"
-    ]
-    otherwise "node"
+  version.default!('6.10.0')
+  requires_when_unmet {
+    on :apt, 'keyed apt source'.with(
+      :uri => 'https://deb.nodesource.com/node_6.x',
+      :release => 'trusty',
+      :repo => 'main',
+      :key_sig => '68576280',
+      :key_uri => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key'
+    )
   }
-  provides {
-    via :apt, "nodejs ~> #{owner.version}"
-    otherwise "node ~> #{owner.version}"
+  installs {
+    via :apt, "nodejs"
+    via :brew, "node"
   }
+  provides "node ~> #{version}"
```

Comment:
will this eventually fail if `version` is fixed to `6.10.0` and nodesource provides a higher `6.x` release?